.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Hey, did you by any chance use `git add .` ? I wondered because this file doesn't look like it belongs here. What is this file for?

Reply:
Sorry, I didn't know that.

Reply:
It's ok, no need to apologise! What part didn't you know? What is this file for?